aws-modules/aws-s3/src/test/java/com/baeldung/s3/FileGeneratorUtil.java

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.s3;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+
+class FileGeneratorUtil {
+
+    private static final String MAGIC_WORD = "baeldung";
+    private static final ThreadLocalRandom RANDOM = ThreadLocalRandom.current();
+
+    public static File generate() {
```

Comment:
What about using Files.createTempFile

Reply:
Yes, that'll be much better. Review incorporated.